components/modal.css

```diff
@@ -1 +1,72 @@
-@import url(../styles.css);
\ No newline at end of file
+@import url(../styles.css);
+
+.modalContainer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 12rem;
+    box-shadow: 1px 1px 10px 8px rgba(0,0,0,0.1);
+    margin: 0 4rem;
+}
+
+.modal-content{
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    justify-content: center;
+    align-items: center;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.4);
+}
+
+.modal{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 40%;
+    left: 40%;
+    background-color: var(--white-color);
+    box-shadow: 1px 1px 10px 8px rgba(0,0,0,0.1);
+    border-radius: 10px;
+    padding: 1rem;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-bottom: 2rem;
```

Comment:
You can just use shortcut padding : 1rem 2rem 2rem 2rem, you won't need to write for each side.

Reply:
will do it.